### DISCUSSION TITLE
taksi

### DISCUSSION THREAD

**Discussion by aysenazselcok** [sonsuzus/sonsuzus#468] [Soru Cevap]
Derslerden sonra n okul çocuğu grubu dışarı çıktı ve doğum gününü kutlamak için Polycarpus'u ziyaret etmeye karar verdi. i'inci grubun s(i) arkadaştan ( 1 ≤  s(i) ≤ 4 ) oluştuğunu ve birlikte Polycarpus'a gitmek istediklerini biliyoruz. Oraya taksiyle gitmeye karar verdiler. Her araba en fazla dört yolcu alabilir. Her gruptaki tüm üyeler aynı taksiye binerse (ancak bir taksi birden fazla grubu alabilir) çocukların en az kaç arabaya ihtiyacı olur?

Giriş
İlk satır tam sayı n ( 1 ≤  n  ≤ 10⁵ ) içerir — okul çocuklarının gruplarının sayısı. İkinci satır bir tam sayı dizisi s (1) ,  s (2) , ...,  s(n) ( 1 ≤  s(i)  ≤ 4 ) içerir. Tam sayılar bir boşlukla ayrılır, s(i) i -inci gruptaki çocuk sayısıdır .

Çıktı
Tüm çocukları Polycarpus'a götürmek için gereken minimum taksi sayısını belirten tek sayıyı yazdırın.

Örnek Girdi ve Çıktı
girdi:
5 
1 2 4 3 3
çıktı:
4

**Comment by aysenazselcok** [comment]
```
n=int(input())
sayilar = list(map(int, input().split()))
toplam=0

for i in sayilar:
    toplam=toplam+i

if(toplam%4!=0):
    print(int(toplam/4)+1)

else:
    print(toplam/4)
    
```

**Reply by sonsuzus** [reply]
Senin çözümünden şüphelendim. Gruplardaki çocukları bölemiyorsun sanırım. O yüzden onları birlikte taksiye koyman gerekir.

**Comment by aysenazselcok** [comment]
Onu nasıl yapmam lazım bilmiyorum

30 Kas 2024 Cmt 20:43 tarihinde Sonsuz Us ***@***.***> şunu
yazdı:
